_pages/emergency-recovery/advanced.md

```diff
@@ -17,3 +17,6 @@ Tails strongly recommended.
 
 #### Verify Validity of Electrum Software
 You can read instructions on how to do this [here](https://www.reddit.com/r/Bitcoin/comments/aa3l6n/how_to_check_if_your_electrum_wallet_is_legit/).
+
+
+{% include next_steps.md %}
```

Comment:
Why did you add this? As-is it does nothing. It's the end of the guide, maybe next steps is back to table of contents?

Reply:
This was just copied from `/emergency-recovery/index.md`. Copying from the related "non-advanced" page is what I did for the other missing "next steps" and I just did the same here, not realizing it didn't really apply!